exporter/clickhousetracesexporter/clickhouse_factory.go

```diff
@@ -60,8 +61,17 @@ func ClickHouseNewFactory(migrations string, datasource string, dockerMultiNodeC
 		TagKeys:     []tag.Key{exporterKey, tableKey},
 		Aggregation: writeLatencyDistribution,
 	}
+	processingTimeDistribution := view.Distribution(100, 200, 300, 400, 500, 600, 700, 800, 900, 1000, 1100, 1200, 1300, 1400, 1500, 1750, 2000, 4000, 8000, 16000, 32000, 64000, 128000, 256000, 512000)
```

Comment:
What's the rationale for this bucket layout?

Reply:
i wanted latencies in the range 0-1.5s to be as accurate as possible so used this
this is just for debugging..it is not meant to be released